assets/app/features/upgrade/i18n/en.json

```diff
@@ -133,6 +133,14 @@
                 "form": {
                     "check-again": "Check"
                 }
+            },
+            "upgrade-admin": {
+                "name": "",
```

Comment:
I'm not using it for anything but it was added like this in many places so I assumed this is some kind of workaround or something... if it's not, I'll remove it